config/routes.rb

```diff
@@ -137,6 +137,8 @@
     path: "email_authentications",
     controllers: {invitations: "email_authentications/invitations"}
 
+  post "email_authentications/validate", to: "email_authentications/password_validations#create"
```

Comment:
I'm open to feedback on the url path. I left "password" out of the url because I didn't see any reason to draw attention to it. Security through obscurity and all.